tools/docker/builder/image-build.sh

```diff
@@ -9,7 +9,7 @@ scriptdir="$(cd "${0%/*}"; pwd)"
 topdir="${scriptdir%/*/*/*/*}"
 
 main() {
-    run docker image build --tag prplmesh-build ${scriptdir}
+    run docker image build --build-arg topdir=$topdir --build-arg user=$SUDO_USER --build-arg uid=$(id $SUDO_USER -u) --tag prplmesh-build ${scriptdir}
```

Comment:
sudo also sets SUDO_UID, so you can use that directly. Also, if my suggestion above works, you don't need to pass user.
```suggestion
    run docker image build --build-arg topdir=$topdir --build-arg uid=$SUDO_UID --tag prplmesh-build ${scriptdir}
```

Also, SUDO_UID should be set if it is not given (e.g. in Travis it won't be, I think). Something like
```shell
SUDO_UID=${SUDO_UID:-0}
```

Reply:
ok